dbt_project.yml

```diff
@@ -774,16 +715,6 @@ seeds:
             evt_block_time: timestamp
             evt_tx_hash: string
             price: string
-        opensea_wyvern_trades_samples:
-          +column_types:
-            tx_hash: string
-            block_time: timestamp
-            token_id: string
-            amount_original: double
-            buyer: string
-            seller: string
-            platform_fee_amount: double
-            royalty_fee_amount: double
```

Comment:
Do you know why this part is moved in this PR? Maybe this is the cause of the issue?
If the PR becomes messy, we can migrate the intended/expected changed files to a new branch, and resubmit/recommit in another PR.

Reply:
i ran into similar issue with piplock & dbt_project.yml file and tried to replace with file in similar manner to Optimism PR. I'm not clear on best way to update those